src/styles/getMuiTheme.js

```diff
@@ -224,4 +245,11 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       borderColor: palette.borderColor,
     },
   }, muiTheme);
+
+  const transformers = computedTheme.transformers ? computedTheme.transformers :
```

Comment:
Haha! Putting the transformers as a function argument, as you suggested, was also my first thought, and I was trying really really hard not to do it :laughing: 

I suppose part of me is really wanting to make sure this feature isn't abused and isn't going to be something 3 months from now we're trying to figure out how to deprecate properly. I know I'm probably being over paranoid, but it makes me feel better if someone and everyone else agrees that it's okay. I'd rather assume what I'm doing is unsafe.